torch/_functorch/_aot_autograd/schemas.py

```diff
@@ -176,7 +177,8 @@ class SubclassCreationMeta:
     # This is needed because we need the autograd metadata on the original subclass
     # (this is guaranteed to be a wrapper subclass that holds a fake tensor,
     #  so holding onto this at runtime shouldn't leak memory)
-    original_subclass: torch.Tensor
+    original_subclass: Optional[torch.Tensor]
```

Comment:
nit: probably worth a quick comment explaining why the type is Optional (actually required at compile time but not needed at runtime so we null it out)